pkgs/development/python-modules/awscrt/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, buildPythonPackage, fetchPypi, cmake, perl, gcc10, mock }:
+
+buildPythonPackage rec {
+  version = "0.11.13";
+  pname = "awscrt";
+
+  buildInputs = [ gcc10 ];
+
+  nativeBuildInputs = [ cmake perl mock ];
```

```suggestion
  nativeBuildInputs = [ cmake gcc perl ];
```

Mock most likely goes into checkInputs.